### DISCUSSION TITLE
Error when compiling the Lammps

### DISCUSSION THREAD

**Discussion by ZhanPengOu** [deepmodeling/deepmd-kit#1045] [Q&A]
Hi,developer
 I met some diffculties that compiling the Lammps,error messages are following:

.....
.....
.....
mpicxx -g -O3   -DLAMMPS_GZIP -DLAMMPS_MEMALIGN=64   -std=c++11 -DHIGH_PREC   -I/home/zhanpengou/deepmd-kit/include -I/home/zhanpengou/deepmd-kit/include -I/home/zhanpengou/deepmd-kit/include/  -DMPICH_SKIP_MPICXX -DOMPI_SKIP_MPICXX=1    -c ../reader_native.cpp
mpicxx -g -O3   -DLAMMPS_GZIP -DLAMMPS_MEMALIGN=64   -std=c++11 -DHIGH_PREC   -I/home/zhanpengou/deepmd-kit/include -I/home/zhanpengou/deepmd-kit/include -I/home/zhanpengou/deepmd-kit/include/  -DMPICH_SKIP_MPICXX -DOMPI_SKIP_MPICXX=1    -c ../modify.cpp
In file included from ../fix_dplr.h:12:0,
                 from ../style_fix.h:15,
                 from ../modify.cpp:17:
../pair_deepmd.h:11:10: fatal error: deepmd/DeepPot.h: No such file or directory
 #include "deepmd/DeepPot.h"
          ^~~~~~~~~~~~~~~~~~
compilation terminated.
make[1]: *** [modify.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory `/home/zpou/lammps-stable_3Mar2020/src/Obj_mpi'
make: *** [mpi] Error 2

it seem that something went wrong in the header files,but i had no idea about it
Looking forward to your reply, thanks for you help

Version:
deepmd-kit-v2.0.0b3

**Comment by njzjz** [comment]
/home/zhanpengou/deepmd-kit/include/deepmd/DeepPot.h

Does this file exist?

**Reply by ZhanPengOu** [reply]
thanks for your reminder, i found that the  user name of cluster is different from my laptop, then i change each of them in the USER-DEEPMD folder , but the next error occur，error messages are following：

.....
.....
...../home/zpou/deepmd-kit/include/tensorflow/core/protobuf/remote_tensor_handle.pb.h: In member function 'std::__cxx11::string* tensorflow::eager::RemoteTensorHandle::unsafe_arena_release_op_device()':
/home/zpou/deepmd-kit/include/tensorflow/core/protobuf/remote_tensor_handle.pb.h:711:21: error: 'struct google::protobuf::internal::ArenaStringPtr' has no member named 'UnsafeArenaRelease'
   return op_device_.UnsafeArenaRelease(&::PROTOBUF_NAMESPACE_ID::internal::GetEmptyStringAlreadyInited(),
                     ^~~~~~~~~~~~~~~~~~
/home/zpou/deepmd-kit/include/tensorflow/core/protobuf/remote_tensor_handle.pb.h: In member function 'void tensorflow::eager::RemoteTensorHandle::unsafe_arena_set_allocated_op_device(std::__cxx11::string*)':
/home/zpou/deepmd-kit/include/tensorflow/core/protobuf/remote_tensor_handle.pb.h:722:14: error: 'struct google::protobuf::internal::ArenaStringPtr' has no member named 'UnsafeArenaSetAllocated'; did you mean 'SetAllocated'?
   op_device_.UnsafeArenaSetAllocated(&::PROTOBUF_NAMESPACE_ID::internal::GetEmptyStringAlreadyInited(),
              ^~~~~~~~~~~~~~~~~~~~~~~
              SetAllocated
make[1]: *** [fix_dplr.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: *** [modify.o] Error 1
make[1]: *** [lammps.o] Error 1
make[1]: Leaving directory `/home/zpou/lammps-stable_3Mar2020/src/Obj_mpi'
make: *** [mpi] Error 2

Could you tell me what should i do next to figure out this problem?
thanks for you help.

**Reply by njzjz** [reply]
You may be using a wrong version of protobuf header. See my comment in #642.
